website/docs/explorer/configuration.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Configuration
+hide_title: true
+---
+
+import TierLabel from "./../_components/TierLabel";
+import AlphaWarning from "../_components/_alpha_warning.mdx";
+
+# Configuration <TierLabel tiers="Enterprise" />
+
+<AlphaWarning/>
+
+This page helps you to understand the options available to configure Explorer
+
+## Prerequisites
+Before using Explorer, please ensure that:
+- You have Weave Gitops Enterprise [v0.21.2](../../releases)
+
+## Setup
+
+Explorer is enabled via configuration through the feature flag  `enableExplorer` that you could
+configure in your Weave Gitops Enterprise helm release values file
+
+```yaml
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: weave-gitops-enterprise
+  namespace: flux-system
+spec:
+  # ... other spec components
+  values:
+    enableExplorer: true
+```
+
+## Configuration
+
+### Clusters
+
+Explorer watches the [GitopsClusters](https://docs.gitops.weave.works/docs/next/cluster-management/managing-existing-clusters/#connect-a-cluster)
+that you have connected to Weave Gitops Enterprise.
+
+### Kinds
+
+Explorer watches for the following kind resources out of the box:
+
+- RBAC: ClusterRole, Role, ClusterRoleBinding and RoleBinding
+- Flux Applications: Kustomizations, HelmRelease
+
+## Data Layer
+
+Explorer take a simple approach to manage resource views. It leverages a Data Store for caching the views and query them.
+The storage lifecycle is bounded to Weave Gitops Enterprise app and does not provide persistence guarantees.
+Instead, it requests data as required to the leaf clusters. In its simplest form, the data store used is [SQLite](https://sqlite.org/index.html).
+
+## Authentication and Authorization
+
+There are two main paths to consider within Explorer in the context of authentication and authorization (authN/authZ):
+
+1. The read or querying path is exercised when a weave gitops user queries the data.
+2. The write or collecting path is exercised to gather the resources from the leaf clusters.
+
+We look into them separately.
+
+## Authentication and Authorization for querying
+
+Explorer leverages existing authentication and authorization built-in the [application](https://docs.gitops.weave.works/docs/configuration/securing-access-to-the-dashboard/).
+It identifies for a user logged in the application: its identity and the access permissions via Kuberentes RBAC.
+Query results are filtered honouring the access determined via RBAC.
+
+## Authentication and Authorization for collecting
+
+As mentioned earlier, [GitopsClusters](https://docs.gitops.weave.works/docs/next/cluster-management/managing-existing-clusters/#connect-a-cluster)
+defines the connection and security context that explorer will use to collect data from the leaf clusters.
+Therefore, you should ensure that the user has access to watch the [Kinds](#kinds) that you would like explorer to serve.
```

Comment:
Is the collector, collecting these objects on a per user basis, or are we talking about the permissions on the `GitOpsClusters` object?  To me this reads that if Person A wants `Kustomizations`, and Person B wants `Kustomizations` and `HelmReleases` then they need to be set up with the "watch" permission?

Reply:
Agree that it is confusing: i have removed it. 

The collection is aync and out of the user context so there should not be reference to the user in this part. 
I do expect that out of this [issue](https://github.com/weaveworks/weave-gitops-enterprise/issues/2732) this section gets extended.